prime-router/src/main/kotlin/azure/ValidateFunction.kt

```diff
@@ -125,8 +125,8 @@ class ValidateFunction(
                         payloadName
                     )
 
-                    // return CREATED status, report submission was successful
-                    HttpStatus.CREATED
+                    // return OK status, report validation was successful
+                    HttpStatus.OK
                 } else HttpStatus.OK
```

Comment:
Why is there an if/else here? both result in HttpStatus.OK. Is the if/else still even valid?  Do the options CheckConnections and ValidatePayload even make sense?  I'd say no, since this *is* the validation endpoint.

Probably just get rid of the if/else entirely. Run the validateAndMoveToProcessing for validation, and then return HttpStatus.OK if it isn't kicked out with an exception by the validateAndMoveToProcessing function

Reply:
@clediggins-usds good call, I've removed it.